Move prover returns: exiting with verification errors
error: abort not covered by any of the `aborts_if` clauses
   ┌─ tests/sources/functional/script_incorrect.move:11:1
   │
11 │ ╭ spec main {
12 │ │     pragma verify = true;
13 │ │     aborts_if false;
14 │ │ }
   │ ╰─^
   │
   ┌─ tests/sources/functional/script_provider.move:15:9
   │
15 │         assert!(signer::address_of(account) == @0x1, 1);
   │         ----------------------------------------------- abort happened here with code 0x1
   │
   =     at tests/sources/functional/script_incorrect.move:7: main
   =         account = <redacted>
   =     at tests/sources/functional/script_incorrect.move:8: main
   =     at tests/sources/functional/script_provider.move:14: register
   =         account = <redacted>
   =     at tests/sources/functional/script_provider.move:15: register
   =     at ../move-stdlib/sources/signer.move:12: address_of
   =         s = <redacted>
   =     at ../move-stdlib/sources/signer.move:13: address_of
   =         result = <redacted>
   =     at ../move-stdlib/sources/signer.move:14: address_of
   =     at tests/sources/functional/script_provider.move:15: register
   =         ABORTED
